src/runtime/runtime.cs

```diff
@@ -147,6 +147,18 @@ public class Runtime
 
         private static PyReferenceCollection _pyRefs = new PyReferenceCollection();
 
+        internal static int pyversionnumber
```

Comment:
Can you use `System.Version` here? `major * 10 + minor` led to compatibility issue in many products before. This is not going to work for 3.10

Reply:
@lostmsu great point.  Updated